templates/documentation.html

```diff
@@ -0,0 +1,297 @@
+              </h3>
+              <div class="panel panel-default">
+                <div class="panel-heading">InvenioILS</div>
+                <div class="list-group">
+                  <a class="list-group-item" href="https://invenioils.docs.cern.ch/quickstart/">
+                    <i class="fa fa-fw fa-play-circle"></i>
+                    Getting started
+                  </a>
+                  <a class="list-group-item" href="https://invenioils.docs.cern.ch/">
+                    <i class="fa fa-fw fa-book"></i>
+                    Documentation
+                  </a>
+                </div>
+              </div>
+              <div class="list-group">
+                <a class="list-group-item" href="#">
+                  <i class="fa fa-fw fa-rocket"></i>
+                  Deploy
+                </a>
+                <a class="list-group-item" href="#">
+                  <i class="fa fa-fw fa-paint-brush"></i>
+                  Advanced customizations
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="col-lg-8 col-lg-offset-2 documentation-search-block">
+          <p class="text-center">
+            Search for anything among all Invenio repositories
+          </p>
+          <div class="input-group">
+            <input type="search" required class="form-control" placeholder="Search for code, documentation, issues, ..."
+                   id="search-query">
+            <div class="input-group-btn">
+              <button type="button" class="btn btn-default btn-search" data-provider="github">
+                <i class="fa fa-fw fa-flip-horizontal fa-search documentation-icon-dark"></i>
+                GitHub
+              </button>
+              <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+                      aria-expanded="false">
+                <span class="caret"></span>
+              </button>
+              <ul class="dropdown-menu dropdown-menu-normal">
+                <li class="dropdown-header">Other</li>
+                <li>
+                  <a href="#" class="btn-search" data-provider="google">
+                    <i class="fa fa-fw fa-flip-horizontal fa-search documentation-icon-dark"></i>
+                    Google
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="documentation-section section-content-white-bg">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-6">
+          <div class="row centered">
+            <span class="section-title">Getting help</span>
+            <div class="col-lg-12">
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-keyboard-o"></i> <a href="https://gitter.im/inveniosoftware/invenio">Chat</a>
```

Comment:
Should we advertise Discord already @lnielsen?

Reply:
Yes, let's do that. We just need to announce it on the Gitter channel......can you take a task on that?

Reply:
Discord invite link successfully tested by @FlorianCassayre , we can replace it here as a first step towards https://github.com/inveniosoftware/opensource/issues/67.